lib/Echidna/Campaign.hs

```diff
@@ -188,10 +217,13 @@ callseq v w ql = do
   coverageEnabled <- isJust . knownCoverage <$> view hasLens
   let ef = if coverageEnabled then execTxOptC else execTx
       old = v ^. env . EVM.contracts
+  -- Reset the new coverage flag
+  hasLens . newCoverage .= False
```

Comment:
can we reset this flag we read it, like [here](https://github.com/crytic/echidna/pull/353/files#diff-a050da14fd68a4b940e8d477c2a28ba3R236)? since we only use it there (i think) it makes sense to have a pre-condition of it being false

Reply:
You mean `when (s ^. _2 . newCoverage) $ hasLens . newCoverage .= False` ?

Reply:
i don't think we need the `when`

Reply:
Fixed.